internal/runners/platforms/remove.go

```diff
@@ -14,19 +14,19 @@ type RemoveRunParams struct {
 
 // Remove manages the removeing execution context.
 type Remove struct {
-	*project.Project
+	prime primeable
```

Comment:
Inconsistent with how we use primeable in runners (handle it in the constructor).